foodsaving/pickups/serializers.py

```diff
@@ -257,24 +257,24 @@ def create(self, validated_data):
 
         History.objects.create(
             typus=HistoryTypus.SERIES_CREATE,
-            group=series.store.group,
-            store=series.store,
+            group=series.place.group,
+            place=series.place,
             series=series,
             users=[
                 self.context['request'].user,
             ],
             payload=self.initial_data,
             after=PickupDateSeriesHistorySerializer(series).data,
         )
-        series.store.group.refresh_active_status()
+        series.place.group.refresh_active_status()
         return series
 
-    def validate_store(self, store):
-        if not store.group.is_editor(self.context['request'].user):
+    def validate_place(self, place):
+        if not place.group.is_editor(self.context['request'].user):
             raise PermissionDenied(_('You need to be a group editor'))
-        if not store.group.is_member(self.context['request'].user):
-            raise serializers.ValidationError(_('You are not member of the store\'s group.'))
-        return store
+        if not place.group.is_member(self.context['request'].user):
+            raise serializers.ValidationError(_('You are not member of the place\'s group.'))
```

Comment:
Translation message

Reply:
I left these ones in as "place" actually. They are these kind of error messages that should not normally be seen...